docs/2.drivers/deno.md

```
@@ -1,76 +1,69 @@
+::note{to="https://deno.com/kv"}  
+了解更多关于 Deno KV 的信息。  
+::  
 
-## Usage (Deno)
+## 使用方法 (Deno)  
 
-::important
-`deno-kv` driver requires [Deno deploy](https://docs.deno.com/deploy/kv/manual/on_deploy/) or [Deno runtime](https://docs.deno.com/runtime/) with `--unstable-kv` CLI flag. See [Node.js](#usage-nodejs) section for other runtimes.
-::
+::important  
+`deno-kv` 驱动需要 [Deno deploy](https://docs.deno.com/deploy/kv/manual/on_deploy/) 或者带有 `--unstable-kv` CLI 标志的 [Deno runtime](https://docs.deno.com/runtime/)。有关其他运行时的请参见 [Node.js](#usage-nodejs) 部分。  
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)